ansible/roles/nginx/files/openwhisk-client-ca-key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
should we perhaps generate these as part of the setup rather that store them.
i know it's convenient and it does take a few seconds to get a key.
since i generated the previous keys, i'm also asking myself if that was the right thing to do and we should avoid that. that way you can see what values are used for the key, and can change them.